### DISCUSSION TITLE
About KERNEL_BANK in scripts\kernel-synth.py

### DISCUSSION THREAD

**Discussion by kkiskkk** [amazon-science/chronos-forecasting#139] [Q&A]
Are there any specific rules for the list of kernel functions in KERNEL_BANK, or are random combinations of kernel functions acceptable just to increase diversity?
Can I use other functions in gpytorch.kernels instead?
Looking forward to your reply, thank you~

**Comment by abdulfatir** [comment]
@kkiskkk As you see in the kernel-synth script (which was used for the paper), there are no specific rules per se. We have some periodic kernels with specific periods denoting common seasonalities, some linear kernels, white noise kernels, etc. You might be able to construct "better" kernels if you enforce some rules on the bank and how you combine the kernels. This is open to further research and experimentation. If you find something, don't forget to share it with everyone. :)
